src/main/java/command/TaskCompletionCommand.java

```diff
@@ -0,0 +1,54 @@
+package command;
+
+import task.TaskList;
+
+public class TaskCompletionCommand extends Command {
+    private boolean isCompleted;
+    private final String[] TASK_COMPLETED_PREFIXES = new String[] {
+        "Nice! Task marked as completed\n",
+        "Nice! Task marked as completed\n",
+        "Nice! Task marked as completed\n",
+        "Alright! Task marked as completed\n",
+        "Phew! Got that one out of the way!\n",
+        "Phew! Got that one out of the way!\n",
+        "Wahoo! Task complete!\n",
+        "Wahoo! Task complete!\n",
+        "Done and done! Should we do it again?\n",
+        "Done and done! Should we do it again?\n"
+    };
+
+    private final String[] TASK_UNCOMPLETED_PREFIXES = new String[] {
+        "Marking as incomplete.\n",
+        "Booooo\n",
```

Comment:
I like this as it makes the chatbot more lively!

Reply:
Thank you!